Cargo.toml

```diff
@@ -37,7 +37,7 @@ rev = "184b2ca50ed39ca43da13f0b830a463861adb9ca"
 [workspace.dependencies.fsrs]
 # version = "3.0.0"
 git = "https://github.com/open-spaced-repetition/fsrs-rs.git"
-rev = "092c20bac7d9239a991ae5b561556ad34c706c16"
+rev = "53784503103169ca6c0702ae2a6d5c84c3d42269"
```

Comment:
Please note that as this was squash merged, that commit may disappear - we need to use the rev from main instead.